### DISCUSSION TITLE
create-custom-decorator.rst does not describe how/where to setup get_provider_info

### DISCUSSION THREAD

**Discussion by henrique** [apache/airflow#32737] [Q&A]
### What do you see as an issue?

The page [create-custom-decorator](https://github.com/apache/airflow/blob/main/docs/apache-airflow/howto/create-custom-decorator.rst) does not describe how or where to place get_provider_info.py

### Solving the problem

https://github.com/apache/airflow/blob/main/docs/apache-airflow/howto/custom-operator.rst is a better example, as it provides pointers for configuring PYTHONPATH and further link to [:doc:`/administration-and-deployment/modules_management`](https://github.com/apache/airflow/blob/main/docs/apache-airflow/howto/custom-operator.rst#id9)

Unfortunately, the same tips don't work for the decorators (at least it seems not to in the standalone version), or should they?

### Anything else

Thank you

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
https://airflow.apache.org/docs/apache-airflow-providers/index.html#how-to-create-your-own-provider

**Comment by potiuk** [comment]
FYI. @henrique  I am improving that part a bit in https://github.com/apache/airflow/pull/32604 to separate out "how to create custom operator" to a new page, and have explicit links to it from those places that were referring to `get_provider_info` without linking to previous section  describing it.
